.github/workflows/build-and-test.yml

```diff
@@ -66,3 +66,7 @@ jobs:
     - name: Build and test P4MLIR
       run: |
         build_tools/build_p4c_with_p4mlir_ext.sh
+    
+    - name: Run P4MLIR tests
+      run: |
+        build_tools/unittest.sh
```

Comment:
If your goal is to change the CI workflow, I'd suggest do it in a separate PR.

Also, I'd like to know why you might want to do this?